samcli/local/docker/container.py

```diff
@@ -344,11 +347,21 @@ def wait_for_result(self, full_path, event, stdout, stderr, start_timer=None):
         # start the timer for function timeout right before executing the function, as waiting for the socket
         # can take some time
         timer = start_timer() if start_timer else None
-        self.wait_for_http_response(full_path, event, stdout)
+        response = self.wait_for_http_response(full_path, event, stdout)
```

Comment:
Should `response` be printed into logs (as a debug)  maybe?  We weren't using it before, but now it is assigned but not used (as far as I can see) 🤔

Reply:
Used on line 357

Reply:
Oh that removal of function definition (`wait_for_logs`) confused me since it has different indentation.